tools/openshift/ocp-ipi-aws/ocp-ipi-aws-prep/ec2-resources.tf

```diff
@@ -61,6 +61,38 @@ resource "aws_security_group_rule" "worker_to_master_sg_vxlan_rule" {
 
 
 
+# Add a rule for metrics traffic for all workers.
+resource "aws_security_group_rule" "worker_sg_metrics_rule" {
+  security_group_id        = data.aws_security_group.worker_sg.id
+  source_security_group_id = data.aws_security_group.worker_sg.id
+  from_port                = 8080
+  protocol                 = "tcp"
+  to_port                  = 8080
+  type                     = "ingress"
+}
+
+# Add a rule for metrics traffic from master nodes to worker nodes.
+resource "aws_security_group_rule" "master_to_worker_sg_metrics_rule" {
+  security_group_id        = data.aws_security_group.worker_sg.id
+  source_security_group_id = data.aws_security_group.master_sg.id
+  from_port                = 8080
+  protocol                 = "tcp"
+  to_port                  = 8080
+  type                     = "ingress"
+}
+
+# Add a rule for metrics traffic from worker nodes to master nodes.
+resource "aws_security_group_rule" "worker_to_master_sg_metrics_rule" {
```

Comment:
This is not required as we do not schedule Submariner-Engine pod on the Master nodes. The gateway-machine that we prepare as part of prep_for_subm.sh script associates the node with the worker_sg.id, so we just have to allow traffic from worker_sg.id and master_sg.id

Reply:
Done, thanks.